hangover.sh

```diff
@@ -1,5 +1,26 @@
 #!/bin/sh -e
 
+# Fill in the blanks
+# Phone numbers with leading country code: +46 012 345 6788
+MY_NUMBER=''
+NUMBER_OF_BOSS=''
+
+# Twilio account information
+TWILIO_ACCOUNT_SID=""
```

Comment:
This will break using environment variables and make it anoying to commit other changes if the values are set.